src/main/java/at/researchstudio/sat/merkmalservice/model/Standard.java

```diff
@@ -0,0 +1,37 @@
+package at.researchstudio.sat.merkmalservice.model;
+
+import java.util.Objects;
+
+public class Standard {
+    private final String id;
```

Comment:
i think we should already add the missing properties:

description
"isPublic"

Reply:
done